test/parallel/test-buffer.js

```diff
@@ -1179,3 +1179,7 @@ Buffer.poolSize = ps;
 assert.throws(function() {
   Buffer(10).copy();
 });
+
+assert.throws(function() {
```

Comment:
Maybe include an explicit test for `null` too, just in case the code changes in the future.